server/shared/types/categorical.ts

```diff
@@ -0,0 +1,26 @@
+import { TermWrapper, BaseQ } from './termdb'
+
+/*
+--------EXPORTED--------
+CategoricalConditionQ
+CategoricalTW
+
+*/
+
+export type CategoricalConditionQ = BaseQ & {
```

Comment:
@creilly This is incorrect and/or confusing. Categorical tw.q should not be mixed in with condition tw.q, only the latter can have bar_by_children, value_by_*.

Reply:
Ahh, I see. I can go through it again and make those changes.